ramanujantools/cmf/cmf_benchmark.py

```diff
@@ -21,6 +21,15 @@ def test_trajectory_matrix_euler(benchmark):
     benchmark(CMF.trajectory_matrix, cmf, trajectory, start)
```

Comment:
lets try to test, 2f2 and 2f1 with deep trajectories, 12,13,14,20.

and also still was slow,
6f5 (z=1) at 1,1,1,1,1,1,2,2,2,2,2.

Reply:
@ElyasheevLeibtag I added the 2f2 deep trajectory test. Unfortunately loading a 6f5 CMF takes too long for now :(